index.html

```diff
@@ -183,23 +183,30 @@ <h3 class="skill-language">Skills</h3>
       </section>
     </section>
 
-    <section class="formx">
-      <div class="contact-form" id="contact">
+    <section class="contact-form">
+      <div class="" id="contact">
+
         <h2 class="contact-title">Contact me</h2>
         <p class="contact-text">If you have an application you are interested in developing, a feature that you need
           built
-          or a project that needs coding. I'd love to help with it </p>
-        <div class="form">
-          <form action="https://formspree.io/f/xbjbnvvz" method="POST">
-            <input class="form-input" type="text" placeholder="Enter your name" maxlength="30" required>
-            <input class="form-input" type="email" placeholder="Enter your Email" required>
-            <textarea class="form-textarea" name="area" cols="30" rows="6" maxlength="500"
-              placeholder="Write your message here" required></textarea>
-            <button class="form-button" type="submit">Get in touch</button>
-          </form>
-        </div>
+          or a project that needs coding.
+          <br> I'd love to help with it
+        </p>
       </div>
+
+      <div class="form-container">
+        <form action="https://formspree.io/f/xrgvywbn" method="POST">
+          <input type="text" name="name" id="name" maxlength="30" required placeholder="Lorem Ipsum">
+          <input type="email" name="email" id="email" required placeholder="loremipsum@gmail.com">
+          <textarea name="message" id="message" maxlength="500" required
+            placeholder="Write your message here..."></textarea>
+          <button class="get-resume get-in-touch" type="submit">Get in touch</button>
```

Comment:
- [ ] Kindly make sure to style the form `input` and the `textarea` in order to match the provided Figma design as much as possible.

- [ ] Kindly make sure to style the `Get in touch` button in order to match the provided Figma design. Kindly implement all button interactions in order to meet the project requirement as well.


  - ## YOUR DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/230600806-756e71a3-8cb4-48db-b3cf-47f53c185e45.png)

  - ### FIGMA DESIGN
  
     ![image](https://user-images.githubusercontent.com/87197412/230600860-c4c8ec59-5d56-4525-9ac1-979051ec806b.png)

  - ### BUTTON INTERACTION DESIGN
  
      ![image](https://user-images.githubusercontent.com/87197412/230600931-17d42630-bdd4-4747-b540-d9b0663500cd.png)